SmartCar.ino

```diff
@@ -188,3 +218,68 @@ void setReverseLeft(){
 void setReverseRight(){
   car.setAngle(135);
 }
+
+//LED lighting modes methods
+
+ void drivingLights(){ // this method turns on the driving mode Lights.
+  yellowCurrent = millis();
+  if(yellowCurrent -previousYellowDrive>=YellowDrivingInterval){
+  digitalWrite(yellowRight, !digitalRead(yellowRight));
+  digitalWrite(yellowLeft, !digitalRead(yellowLeft));
+  previousYellowDrive = yellowCurrent;  
+ }
+  
+ redCurrent = millis();
+ if(redCurrent -previousRedDrive>=redDrivingInterval){
+  digitalWrite(redRight, !digitalRead(redRight));
+  digitalWrite(redLeft, !digitalRead(redLeft));
+  previousRedDrive = redCurrent;
+ }
+ }

```

Comment:
If you press Ctrl+t your Arduino IDE will automatically indent the code and will look prettier. :+1: